src/PowerShellEditorServices/Workspace/ScriptFile.cs

```diff
@@ -650,7 +650,7 @@ private void ParseFileContents()
 
             //Get all dot sourced referenced files and store  them
             this.ReferencedFiles =
-                AstOperations.FindDotSourcedIncludes(this.ScriptAst);
+                AstOperations.FindDotSourcedIncludes(this);
```

Comment:
I know this is the original style, but while you're here, maybe put this statement back on the same line as the assignment